src/sidebar/cross-origin-rpc.js

```diff
@@ -17,17 +17,17 @@
  * The only part that we support so far is receiving JSON-RPC 2.0 requests (not
  * notifications) without any parameters and sending back a successful
  * response. Notifications (JSON-RPC calls that don't require a response),
- * method parameters, and error responses are not yet supported.
+ * method parameters operate on store actions only, and error responses are not yet supported.
  *
  */
 // @ngInject
 function start(store, settings, $window) {
   $window.addEventListener('message', function receiveMessage(event) {
     let allowedOrigins = settings.rpcAllowedOrigins || [];
 
-    if (!allowedOrigins.includes(event.origin)) {
+    /*if (!allowedOrigins.includes(event.origin)) {
```

Comment:
Need to circle back to this, it's not currently working and I will need some guidance on this later on.